lib/node_modules/@stdlib/number/float32/base/signbit/src/addon.c

```diff
@@ -0,0 +1,22 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/signbitf.h"
+#include "stdlib/math/base/napi/unary.h"
+
+STDLIB_MATH_BASE_NAPI_MODULE_D_B( stdlib_base_signbitf )
```

Comment:
@kgryte So `STDLIB_MATH_BASE_NAPI_MODULE_D_B` doesn't exist in `stdlib/math/base/napi/unary.h`. What should this file look like?

Reply:
There won't be this macro, as it is not common enough. See https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/number/float64/base/signbit/src/addon.c. Unless this pattern happens over and over, not likely to add. As mentioned elsewhere, cannot really predict what should go in `addon.c` as it will vary from package to package, even if for some types of packages, such as `math/base/special/*` where we predominantly use macros, this won't be true elsewhere (e.g., in `number/base/*` or `blas/*` or `strided/*`).

In short, in terms of scaffolding, I'd template the `addon.c` file with a TODO.